matchbox_socket/src/lib.rs

```diff
@@ -3,13 +3,14 @@
 #![forbid(unsafe_code)]
 
 mod error;
-#[cfg(feature = "ggrs-socket")]
-mod ggrs_socket;
+mod integrations;
 mod webrtc_socket;
 
 pub use error::Error;
+pub use integrations::*;
```

Comment:
This puts all the public api in bevy_plugin.rs at the root level? I'm not sure we want that... Maybe better to have everything bevy related in its own module, so people don't get overwhelmed by all the types available at the root (when reading the docs)?